packages/docbook_xsl.rb

```diff
@@ -5,66 +5,50 @@
-            
-            ln -s VERSION #{CREW_DEST_DIR}/usr/local/share/xml/docbook/xsl-stylesheets-1.79.1/VERSION.xsl &&
-            
-            install -v -m644 -D README \
-                                #{CREW_DEST_DIR}/usr/local/share/doc/docbook-xsl-1.79.1/README.txt &&
+
+    xsl_stylesheets = 'xsl-stylesheets-1.79.1'
+    docbook_xsl_ = 'docbook-xsl-1.79.1'
+    xsl_version = 'xsl/1.79.1'
+
+    system "install -v -m755 -d #{CREW_DEST_PREFIX}/share/xml/docbook/#{xsl_stylesheets} &&
+            cp -v -R . \
+                #{CREW_DEST_PREFIX}/share/xml/docbook/#{xsl_stylesheets}/"
+    system "install -v -m644 -D README \
+                                #{CREW_DEST_PREFIX}/share/doc/#{docbook_xsl_}/README.txt &&
             install -v -m644    RELEASE-NOTES* NEWS* \
-                                #{CREW_DEST_DIR}/usr/local/share/doc/docbook-xsl-1.79.1"
-    
+                                #{CREW_DEST_PREFIX}/share/doc/#{docbook_xsl_}"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/xml"
     system "xmlcatalog --noout --add 'rewriteSystem' \
-                       'http://docbook.sourceforge.net/release/xsl/1.79.1' \
-                       '#{CREW_DEST_DIR}/usr/local/share/xml/docbook/xsl-stylesheets-1.79.1' \
-                /usr/local/etc/xml/catalog &&
+                       'http://docbook.sourceforge.net/release/#{xsl_version}' \
+                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
+                #{CREW_PREFIX}/etc/xml/catalog &&
             
             xmlcatalog --noout --add 'rewriteURI' \
-                       'http://docbook.sourceforge.net/release/xsl/1.79.1' \
-                       '#{CREW_DEST_DIR}/usr/local/share/xml/docbook/xsl-stylesheets-1.79.1' \
-                /usr/local/etc/xml/catalog &&
+                       'http://docbook.sourceforge.net/release/#{xsl_version}' \
+                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
+                #{CREW_PREFIX}/etc/xml/catalog &&
             
             xmlcatalog --noout --add 'rewriteSystem' \
                        'http://docbook.sourceforge.net/release/xsl/current' \
-                       '#{CREW_DEST_DIR}/usr/local/share/xml/docbook/xsl-stylesheets-1.79.1' \
-                /usr/local/etc/xml/catalog &&
+                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
+                #{CREW_PREFIX}/etc/xml/catalog &&
             
             xmlcatalog --noout --add 'rewriteURI' \
                        'http://docbook.sourceforge.net/release/xsl/current' \
-                       '#{CREW_DEST_DIR}/usr/local/share/xml/docbook/xsl-stylesheets-1.79.1' \
-                /usr/local/etc/xml/catalog"
-
+                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
+                #{CREW_PREFIX}/etc/xml/catalog"
+    system "mv #{CREW_PREFIX}/etc/xml/catalog #{CREW_DEST_PREFIX}/etc/xml/"
```

Comment:
Don't we want to create the catalog in a single package instead of 2 separate packages that can tread on each other?

Reply:
No, the catalog for docbook_xml will only ever be in docbook_xml and the catalog for docbook_xsl will only ever be in docbook_xsl.

Reply:
Besides, their needs are different.

Reply:
Unless I'm looking at this wrong, `#{CREW_PREFIX}/etc/xml/catalog` is copied in 2 separate packages.

Reply:
Exactly. The XSL and XML-enabled catalog is in docbook_xsl, while the XML-only catalog is in docbook_xml.

Reply:
The binaries won't work for either packages if the catalogs aren't tracked in both.